pkg/apis/pipeline/v1beta1/pipeline_validation.go

```diff
@@ -208,8 +208,8 @@ func validateExecutionStatusVariablesInTasks(tasks []PipelineTask) (errs *apis.F
 					for _, p := range ps {
 						// check if it contains context variable accessing execution status - $(tasks.taskname.status)
```

Comment:
we need to update this comment, and others above it in this function, to include the aggregate status

Reply:
thanks @jerop I have updated the comments 🙏